tasks/check-data-keys/README.md

```diff
@@ -14,6 +13,10 @@ Currently, `releaseNotes`, and `cdn` are the only supported systems.
 |----------|---------------------------------------------------------|----------|---------------|
 | dataPath | Path to the JSON string of the merged data to use       | No       |               |
 | systems  | The systems to check that all data keys are present for | Yes      | []            |
+| schema   | The URl to the schema                                   | Yes      | https://github.com/konflux-ci/release-service-catalog/blob/production/schema/dataKeys.json |
+
+## Changes in 1.0.0
+* Adding a schema validater to the end of the task
```

Comment:
More like replace the check with a schema validator

Reply:
Fixed